teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization)
+    super(age)
+    @specialization = specialization
+  end
+
```

Comment:
- [ ] In the `initialize` method of the `Teacher` class, you need to pass in the `name` and `parent_permission` data as parameters. Since the `Teacher` class is inherited from the `Person` class, you need to pass in the `name` and `parent_permission` data as parameters of the `super` method. 👍 😎 

#### Hint:

```ruby
def initialize(age, specialization, name = 'Unknown', parent_permission = true)
  super(age, name, parent_permission: parent_permission)
  @specialization = specialization
end
```